java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/syntaxexamples/Example56_TemplateMethodFunctionsWithMultipleArguments.java

```diff
@@ -0,0 +1,88 @@
+package com.microsoft.semantickernel.samples.syntaxexamples;
+
+import com.azure.ai.openai.OpenAIAsyncClient;
+import com.azure.ai.openai.OpenAIClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.credential.KeyCredential;
+import com.microsoft.semantickernel.Kernel;
+import com.microsoft.semantickernel.chatcompletion.ChatCompletionService;
+import com.microsoft.semantickernel.orchestration.PromptExecutionSettings;
+import com.microsoft.semantickernel.orchestration.contextvariables.KernelArguments;
+import com.microsoft.semantickernel.plugin.KernelPluginFactory;
+import com.microsoft.semantickernel.samples.plugins.text.TextPlugin;
+import com.microsoft.semantickernel.semanticfunctions.KernelFunctionFromPrompt;
+import com.microsoft.semantickernel.semanticfunctions.KernelPromptTemplateFactory;
+import com.microsoft.semantickernel.semanticfunctions.PromptTemplateConfig;
+
+public class Example56_TemplateMethodFunctionsWithMultipleArguments {
+
+    private static final boolean USE_AZURE_CLIENT = Boolean.parseBoolean(
+        System.getenv("USE_AZURE_CLIENT"));
+    private static final String CLIENT_KEY = System.getenv("CLIENT_KEY");
+
+    // Only required if USE_AZURE_CLIENT is true
+    private static final String CLIENT_ENDPOINT = System.getenv("CLIENT_ENDPOINT");
+
+    public static void main(String[] args) {
+
+        OpenAIAsyncClient client;
+
+        if (USE_AZURE_CLIENT) {
+            client = new OpenAIClientBuilder()
+                .credential(new AzureKeyCredential(CLIENT_KEY))
+                .endpoint(CLIENT_ENDPOINT)
+                .buildAsyncClient();
+        } else {
+            client = new OpenAIClientBuilder()
+                .credential(new KeyCredential(CLIENT_KEY))
+                .buildAsyncClient();
+        }
+
+        ChatCompletionService openAIChatCompletion = ChatCompletionService.builder()
+            .withOpenAIAsyncClient(client)
+            .withModelId("gpt-35-turbo-2")
```

Comment:
Isn't gpt-35-turbo-2 the name you gave your model deployment? I think this should be an env var which defaults to 'gpt-3.5-turbo' for openai or 'gpt-35-turbo' for azure openai